src/Integration.UnitTests/Helpers/AbsoluteFilePathLocatorTests.cs

```diff
@@ -36,20 +36,31 @@ public class AbsoluteFilePathLocatorTests
     {
         [TestMethod]
         public void MefCtor_CheckIsExported()
+            => MefTestHelpers.CheckTypeCanBeImported<AbsoluteFilePathLocator, IAbsoluteFilePathLocator>(
+                MefTestHelpers.CreateExport<IVsUIServiceOperation>());
+
+        [TestMethod]
+        public void MefCtor_CheckIsSingleton()
+            => MefTestHelpers.CheckIsSingletonMefComponent<AbsoluteFilePathLocator>();
+
+        [TestMethod]
+        public void MefCtor_DoesNotCallAnyServices()
         {
-            var serviceProvider = new Mock<IServiceProvider>();
```

Comment:
The old MEF test was ugly because the constructor called the service provider.

It now has the normal set of MEF tests.